### DISCUSSION TITLE
How to enable dark mode?

### DISCUSSION THREAD

**Discussion by nahasco** [shadcn-ui/ui#616] [Q&A]
Should an easy answer but I could not find anything about this in the docs.

**Comment by YodaheaD** [comment]
I think because it is only in the dashboard component example. Ill try to help. 

Heres link to shadcn's component that toggles the theme "github.com/shadcn/ui/blob/main/templates/next-template/components/theme-toggle.tsx". 

The dark and light theme JSON files can also be found under "ui>apps>www>lib>themes>dark/light.json". Hopefully that helps.

**Comment by oninoor** [comment]
It is updated on the official documentation for adding dark mode to your site [here](https://ui.shadcn.com/docs/dark-mode)

**Comment by eruens** [comment]
shadcn/ui dark mode works with next-themes. Install next-themes, wrap your layout in ThemeProvider with attribute='class', and use useTheme() hook to toggle. Full guide: https://ui.shadcn.com/docs/dark-mode/next
